packages/cdk-cli-wrapper/.warnings.jsii.js

```diff
@@ -0,0 +1,73 @@
+function cdk_cli_wrapper_ICdk(p) {
```

Comment:
Looks like cdk-cli-wrapper folder was committed by mistake?

Reply:
Yeah, I commited this by mistake. Good spot! Thanks